tick/array/build/__init__.py

```diff
@@ -1 +1,7 @@
 # License: BSD 3 clause
+
+import os, platform
+
+if platform.system() == 'Windows':
+  if os.path.dirname(os.path.realpath(__file__)) not in os.environ["PATH"]:
+    os.environ["PATH"] = os.path.dirname(os.path.realpath(__file__)) + os.pathsep + os.environ["PATH"]
```

Comment:
Do you think this code can be put in an external function? Or it will be problematic to import?

Reply:
it's possible - the important part is that "__file__" part